.github/workflows/ci.yaml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
Are you sure you want to run this on any push to **any branch** ?
shouldn't it be just `development` and `master` ?

Reply:
CI ?
yeah, there's no problem of people pushing too much to upstream branches in this repo to make this a load issue on github action servers